### ISSUE TITLE
[8.0] decimal_precison not configurable by company as module describes

### ISSUE THREAD

**Issue by colmeirin** [odoo/odoo#12654]
Impacted versions: 8.0

Steps to reproduce: Install decmial_precision module. Go to Settings / Technical  / Database Structure / Decimal Accuracy and change any value.

Current behavior: It applies the change for all the companies

Expected behavior: Be able to configure it by company or edit the module description to remove the text "The decimal precision is configured per company"

Video/Screenshot link (optional):
![decimal_precision_screenshot](https://cloud.githubusercontent.com/assets/5647643/16516966/476851ca-3f7b-11e6-9906-ef30df568585.png)

**Comment by Yenthe666** [comment]
@mart-e since you're from documentation. Perhaps we could rewrite that one?

**Comment by mart-e** [comment]
The description of this module is indeed wrong. The decimal precision is at the field level so there is technically no way to make it company dependant, even if we added a company field (it is computed when loading the server, not when a user access it).

The field Monetary kind of give you this, as the precision will be now per currency. Unfortunately there is no way to have this on classical float fields without big changes in the fields structure.

Changed the description of the module in master at f71125853594c6afb1472c35b584849832e456f4
